service/src/main/resources/application.conf

```diff
@@ -0,0 +1,590 @@
+SunbirdMWSystem {
+  default-dispatcher {
+    type = "Dispatcher"
+    executor = "fork-join-executor"
+    fork-join-executor {
+      parallelism-min = 8
+      parallelism-factor = 32.0
+      parallelism-max = 64
```

Comment:
delete this file , not needed any more as we moved this under controller/application.conf

Reply:
done